x/cork/types/msgs.go

```diff
@@ -8,10 +8,12 @@ import (
 
 var (
 	_ sdk.Msg = &MsgSubmitCorkRequest{}
+	_ sdk.Msg = &MsgScheduleCorkRequest{}
 )
 
 const (
-	TypeMsgSubmitCorkRequest = "cork_submit"
+	TypeMsgSubmitCorkRequest   = "cork_submit"
+	TypeMsgScheduleCorkRequest = "cork_submit"
```

Comment:
You've reused the same string as `TypeMsgSubmitCorkRequest` here.

Reply:
Sorry, forgot to push